src/Command/UpdateCdnStatusCommand.php

```diff
@@ -68,53 +86,57 @@ public function execute(InputInterface $input, OutputInterface $output)
         $provider = $this->getProvider();
         $context = $this->getContext();
 
-        $medias = $this->getMediaManager()->findBy([
+        $media = $this->getMediaManager()->findBy([
             'providerName' => $provider->getName(),
             'context' => $context,
             'cdnIsFlushable' => true,
         ]);
 
-        $this->log(sprintf('Loaded %s medias for updating CDN status (provider: %s, context: %s)', \count($medias), $provider->getName(), $context));
+        $this->log(sprintf('Loaded %s media for CDN status update (provider: %s, context: %s)', \count($media), $provider->getName(), $context));
 
-        foreach ($medias as $media) {
+        foreach ($media as $medium) {
```

Comment:
I would keep the old wording, because the entity is called `Media`

Reply:
The variable renaming was reverted.